src/components/mdc/Button/_index.scss

```diff
@@ -1,3 +1 @@
-@use '@material/button';
-
-@include button.core-styles;
+@use "@material/button/styles";
```

Comment:
yes, better...

Reply:
will be a mute point after installing @silintnl/ui-components, but thought I'd better anyways.